tests/Support/SupportStrTest.php

```diff
@@ -271,6 +290,25 @@ public function testStrAfter()
         $this->assertSame('nah', Str::after('han0nah', '0'));
         $this->assertSame('nah', Str::after('han0nah', 0));
         $this->assertSame('nah', Str::after('han2nah', 2));
+        $this->assertSame('hannah', Str::after('hannah', 'HAN'));
```

Comment:
Added a case sensitive test to the previous functionality to show that it does not match